rust-toolchain.toml

```diff
@@ -4,4 +4,4 @@ targets = [ "riscv32imc-unknown-none-elf", "riscv32imac-unknown-none-elf",
             "thumbv6m-none-eabi", "thumbv7em-none-eabihf", 
             "thumbv8m.main-none-eabihf" ]
 profile = "minimal"
```

Comment:
Would it make sense to instead change the profile to `default` or do we not want the `rust-docs` component?

Reply:
While a good amount of the code uses rustdoc formatting, only `xtask`'s documentation can be built as `cargo doc` partially compiles the crate too and expects some environment variables to be set. Without an `xtask` function for it there's no way to build the docs for most of the codebase, so I think `minimal` is fine until that gets added.

Reply:
Ack.